AsaApi/Core/Public/API/ARK/Inventory.h

```diff
@@ -969,7 +969,7 @@ struct UPrimalInventoryComponent : UActorComponent
     bool LoadAdditionalStructureEngrams() { return NativeCall<bool>(this, "UPrimalInventoryComponent.LoadAdditionalStructureEngrams()"); }
     bool RemoveItem(const FItemNetID* itemID, bool bDoDrop, bool bSecondryAction, bool bForceRemoval, bool showHUDMessage) { return NativeCall<bool, const FItemNetID*, bool, bool, bool, bool>(this, "UPrimalInventoryComponent.RemoveItem(FItemNetID&,bool,bool,bool,bool)", itemID, bDoDrop, bSecondryAction, bForceRemoval, showHUDMessage); }
     ADroppedItem* EjectItem(const FItemNetID* itemID, bool bPreventImpule, bool bForceEject, bool bSetItemLocation, const UE::Math::TVector<double>* LocationOverride, bool showHUDMessage, TSubclassOf<ADroppedItem> TheDroppedTemplateOverride, bool bAssignToTribeForPickup, int AssignedTribeID) { return NativeCall<ADroppedItem*, const FItemNetID*, bool, bool, bool, const UE::Math::TVector<double>*, bool, TSubclassOf<ADroppedItem>, bool, int>(this, "UPrimalInventoryComponent.EjectItem(FItemNetID*,bool,bool,bool,UE::Math::TVector<double>*,bool,TSubclassOf<ADroppedItem>,bool,int)", itemID, bPreventImpule, bForceEject, bSetItemLocation, LocationOverride, showHUDMessage, TheDroppedTemplateOverride, bAssignToTribeForPickup, AssignedTribeID); }
-    bool ServerEquipItem(FItemNetID& itemID) { return NativeCall<bool, FItemNetID&>(this, "UPrimalInventoryComponent.ServerEquipItem(FItemNetID&)", itemID); }
+    bool ServerEquipItem(FItemNetID* itemID, AShooterPlayerController* RequestSPC) { return NativeCall<bool, FItemNetID*, AShooterPlayerController*>(this, "UPrimalInventoryComponent.ServerEquipItem(FItemNetID&,AShooterPlayerController*)", itemID, RequestSPC); }
```

Comment:
IDA always shows references as pointers in the pseudo-C disassembly (references do not exist in C)
<img width="234" alt="{530B98F4-6D42-4934-95BF-2C82C6C7BAD5}" src="https://github.com/user-attachments/assets/24cc2a30-33d8-4587-8e51-56c2fe3eb382">

The PDB types show FItemNetID should be a reference (like how it used to be)
<img width="260" alt="{EFBC4C0F-C243-4E76-AA2A-C3ABE510D364}" src="https://github.com/user-attachments/assets/c677c9b8-0fb4-4ccc-86f4-1a1be7a9f27e">


Please update `FItemNetID*` to `FItemNetID&` and I'll approve it.